docs/user-process.model.md

```diff
@@ -0,0 +1,26 @@
+# User Process model
+
+Represents a Process Services user.
+
+## Details
+
+This class contains basic information about a Process Services user and
+is used throughout ADF to identify and list users (eg, to assign them to
+a task or to list them in search results).
+
+```ts
+class UserProcessModel implements LightUserRepresentation {
```

Comment:
I prefer to not have a copy of the model in the doc could be easily go out of sync